src/cmd/ship.go

```diff
@@ -158,9 +158,7 @@ func getCanShipWithDriver(branch, parentBranch string) bool {
 		return false
 	}
 	canMerge, err := driver.CanMergePullRequest(branch, parentBranch)
-	if err != nil {
-		log.Fatalln(err)
-	}
+	logs.FatallnOn(err)
```

Comment:
Can we update this to `logs.FatalOn`? In a test, I see zero difference between using `log.Fatal` and `log.Fatalln`. Both output
```
2017/09/01 13:12:56 ...
exit status 1
```
Then we can remove `log.FatallnOn`